dask/tests/test_base.py

```diff
@@ -283,6 +284,52 @@ def test_tokenize_base_types(x):
     assert tokenize(x) == tokenize(x), x
 
 
+@pytest.mark.parametrize('cls_name',
+                         ('ndarray', 'matrix',
```

Comment:
The implementation looks good to me, but this test is more complicated than it needs to be. We already have tests for `np.ndarray`, would be cleaner if you split out the testing for `np.matrix`, and had this test just focus on `scipy.sparse` objects.

Reply:
You are right, thanks for the review. This should be fixed now.